### PR TITLE
chore: passwords API on cloud not supported

### PR DESCRIPTION
**Pull request by wbaker85**
Closes https://github.com/influxdata/idpe/issues/15092

Changing passwords is not supported through the InfluxDB Cloud API, and this PR updates the API docs for that.

### CODE REVIEW

File: src/common/paths/me_password.yml [hunk 1]
```diff
@@ -3,23 +3,31 @@ put:
   tags:
     - Users
   summary: Update a password
+  description: |
+    #### InfluxDB Cloud
+
+    - Doesn't support changing user passwords through the API.
```
Comment (sanderson) [inline src/common/paths/me_password.yml:9]:
```suggestion
    InfluxDB Cloud does not support changing user passwords through the API.
    Use the InfluxDB Cloud user interface to update your password.
```

File: src/common/paths/users_userID_password.yml [hunk 2]
```diff
@@ -4,10 +4,14 @@ post:
     - Security and access endpoints
     - Users
   summary: Update a password
+  description: |
+    #### InfluxDB Cloud
+
+    - Doesn't support changing user passwords through the API.
```
Comment (sanderson) [inline src/common/paths/users_userID_password.yml:10]:
```suggestion
    InfluxDB Cloud does not support changing user passwords through the API.
    Use the InfluxDB Cloud user interface to update your password.
```

**Review by sanderson — CHANGES_REQUESTED** [review]
Some minor suggestions and alternate calls to action.

File: src/common/paths/me_password.yml (hunk 1 above)
Comment (wbaker85) [reply src/common/paths/me_password.yml]: Done, thanks!

**Review by sanderson — APPROVED** [review]

**Review by SimonRichardson — APPROVED** [review]

**Pull request merged**